bindings/python/test/flight/profile/model/test_tabulated_profile.py

```
@@ -63,6 +85,47 @@ def test_constructors(
+                AngularVelocity.default(): Interpolator.Type.BarycentricRational,
+            },
+        )
+
+        assert tabulated_model.is_defined()
+
+    def test_constructor_with_interpolation_types_failure(
+        self,
+        states: list[State],
+    ):
+        # A coordinate subset present in the (reduced) states but missing from the map must raise.
+        with pytest.raises(Exception):
```

Ruff (0.15.15)

[warning] Do not assert blind exception: `Exception`

(B017)